crates/macros/src/lib.rs

```diff
@@ -712,7 +712,7 @@ fn php_module_internal(args: TokenStream2, input: TokenStream2) -> TokenStream2
 /// - `#[php(optional = i)]` - Sets the first optional parameter. Note that this
 ///   also sets the remaining parameters as optional, so all optional parameters
 ///   must be a variant of `Option<T>`.
-/// - `#[php(public)]`, `#[php(protected)]` and `#[php(private)]` - Sets the
+/// - `#[php(vis = "public")]`, `#[php(vis = "protected")]` and `#[php(vis = "private")]` - Sets the
```

Comment:
Don't update these docs manually please.

Change it in the guides and use the script in `tools` to update these docs.

Reply:
I need help with this script, i try to run 
```
./tools/update_lib_docs.sh
```
And its change to old not working variant.
So i decently doing something wrong